data_structures/binary_trees/binarytree_approach2.c

```diff
@@ -0,0 +1,270 @@
+/*
+@file 
+@brief have compelete implementation of a binary search tree
+@include all function defined 
+including insert,find,findMax,delet_tree,delete(to delete a node),preOrder and height.
+which is required for compelete implementation of a tree
```

Comment:
An `* ` is needed before any word is added in all comment blocks.
```suggestion
 * @file 
 * @brief have complete implementation of a binary search tree
 * @include all function defined 
 * including insert, find, findMax, delete_tree, delete(to delete a node), preOrder and height.
 * which is required for complete implementation of a tree
 *
 * @author [Aarti Singh](https://github.com/Aarti002)
```

Reply:
i didnt get it ,how to make seperate PR for each file. :/

Reply:
Make a PR per the files you've edited in this PR.